src/server/handlers/security-handler.ts

```diff
@@ -0,0 +1,9 @@
+import type { Response } from "express";
+
+export default async ({ res }: { res: Response }) => {
+  res.setHeader("content-type", "text/plain; charset=utf-8");
+
+  res.send(`Contact: mailto:security@lemmy.ml
```

Comment:
Ideally this would be driven by the URL you run Lemmy on, or a configuration option.

Reply:
True, however this is literally the first time I've ever done anything with typescript. Although, if you have code I can add in, I'll gladly do so.

Reply:
`process.env.LEMMY_UI_HOST` should work for the instance URL.

Reply:
I can add it as an addition contact but I have concerns security@`process.env.LEMMY_UI_HOST` or admin@`process.env.LEMMY_UI_HOST` will reach the owner of the instance.

Reply:
Added